### DISCUSSION TITLE
Radeon ProRender standalone for Maya

### DISCUSSION THREAD

**Discussion by Habaybeh** [GPUOpen-LibrariesAndSDKs/RadeonProRenderMayaPlugin#255] [Q&A] [answered]
Hi everyone,

is it possible to (somehow) modify this engine to be compiled as a standalone renderer, instead of a plugin on Maya? The purpose is to use the standalone renderer on a remote computer. What's the complexity of such thing?

Thank you.

**Comment by bsavery** [comment, marked as answer]
So is your question about RPR in general?  We do have the RPR SDK available (with examples) that can be compiled to run independent.  https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderSDK/tree/master/tutorials

Or are you looking to export data from Maya and then render in some standalone?

**Reply by Habaybeh** [reply]
I'm looking to export data from Maya and use it on RPR but on a remote location (Cloud).

Can I do that by compiling the SDK you provided as a standalone RPR for Maya?

And thank you.

**Reply by bsavery** [reply]
So that's a bit even easier.  You can export a .rpr file from Maya using the export menu, then use the RPRSRender64.exe included in the SDK.  From here:  https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderSDK/tree/master/RadeonProRender/binWin64
Instructions for using the command line exe:  https://radeon-pro.github.io/RadeonProRenderDocs/en/sdk/command_line_tools.html

**Reply by Habaybeh** [reply]
That sound great. I will give it a go, I hope I can achieve something :)
Thank you so much.
